### ISSUE TITLE
[Bug][DropDownList] Missing Accessibility attributes in the popup

### ISSUE THREAD

**Issue by vveesseelliinnaa** [telerik/kendo-vue#603]
**Describe the bug**
There are missing accessibility attributes in the popup of the component:

E.g:
![image](https://github.com/user-attachments/assets/d1f12048-822e-448f-8e24-54f13489c595)

**To Reproduce**
1. Go to https://www.telerik.com/kendo-vue-ui/components/dropdowns/dropdownlist/filtering/
2. Inspect the rendering of the popup

**Expected behavior**

**Screenshots**
Check the rendering of the Angular DropDownList popup - https://www.telerik.com/kendo-angular-ui/components/dropdowns/dropdownlist/filtering/

![image](https://github.com/user-attachments/assets/b10ceb73-3609-4ee3-a269-dbfd2a98ed96)

**Additional details**
Reported in Ticket ID: 1660500
